backend/src/test/java/com/github/mysterix5/vover/cloudstorage/CloudRepositoryTest.java

```diff
@@ -0,0 +1,38 @@
+package com.github.mysterix5.vover.cloudstorage;
+
+import com.github.sardine.Sardine;
+import org.junit.jupiter.api.Test;
+
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CloudRepositoryTest {
+
+    @Test
+    void find() throws IOException {
+        Sardine mockedSardine = Mockito.mock(Sardine.class);
+                String url = "https://cloud.com/";
+        CloudRepository cloudRepository = new CloudRepository(mockedSardine, url);
+
+        var obFile = new File("src/test/resources/com.github.mysterix5.vover.cloudstorage/ob.wav");
+        var obStream = new FileInputStream(obFile);
+
+        Mockito.when(mockedSardine.get(url + "ob.wav")).thenReturn(obStream);
+        byte[] actual = cloudRepository.find("ob.wav");
+
+        obStream = new FileInputStream(obFile);
```

Comment:
why the duplicate of setting `obStream`?

Reply:
When cloudRepository.find is executed it reads the stream and obStream is empty afterwards. I recreate it to compare the results with assertj